spec/lib/active_merchant/billing/affirm_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe ActiveMerchant::Billing::Affirm do
+
+  let(:gateway_options) do
+    {
+      public_api_key: "XPSQ3CA7PLN7CJCK",
+      private_api_key: "w9mxkQUryKjTYDqOfSvYTeTGoLIURKpU",
+      test_mode: true
+    }
+  end
+
+  let(:affirm_payment_source) { create(:affirm_checkout, token: "MFLJS57G6BT5OQXN") }
```

Comment:
prefer `FactoryGirl.create` when using it instead of the mixed in Variant, which afaik is deprecated or at the least shouldn't be used.